fb-setup-umask.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+
+@test "configure default umask" {
+  if tIsDebianCompatible; then
+    if ! grep -q pam_umask /etc/pam.d/common-session; then
```

Comment:
Oh, let me revisit this tomorrow before merging.  I forgot that both common-session and common-session-noninteractive (on 14.04, at least), need to be updated.